app/models/request.rb

```diff
@@ -51,17 +52,22 @@ class Request < ActiveRecord::Base
     self.item_title ||= bib_data&.title
   end
 
+  after_find do
+    # This converts stored records so that they have a location set.
+    self.location ||= FolioLocationMap.folio_code_for(library_code: origin, home_location: origin_location)
+  end
```

Comment:
could we just do this with a migration?

Reply:
Yes, but I don't like doing data migrations in automated database migrations.  It's possible we get through 90 pct of the records and find something that blocks the migration (perhaps a location that isn't in the migration table.), so I think we should do that apart from this code change.

Reply:
is the general process that you just delete this line after some time has passed? how do we know that everything has been updated?

Reply:
Yes, after you do `Request.find_each { |r| r.save! }`, then all the data is migrated, then we can remove this line.